common/util/BUILD

```diff
@@ -42,6 +42,7 @@ java_library(
     deps = [
         "//intellij_platform_sdk:jsr305",  # unuseddeps: keep for @Nullable
         "//intellij_platform_sdk:plugin_api",
+	"@error_prone_annotations//jar",
     ],
```

Comment:
I think that there is a formatting problem in here. I think that you switched between tabs and spaces or something.